src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,40 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    list-style-type: none;
+    margin: 0;
+  }
+
+  &__element:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    text-decoration: none;
+    color: $color-dark-blue;
+
+  }
+
+  .is-active {
+    color: $color-light-blue;
+    position: relative;
+  }
+
+  .is-active::after {
```

Comment:
You can write it inside `.is-active`

Reply:
I probably should. Must have missed one when was refactoring nav.